### DISCUSSION TITLE
RenderTexture is not working in  pixi v 7.0.5

### DISCUSSION THREAD

**Discussion by ajith-1998** [pixijs/pixijs#9017] [Q&A]
I have created a rendertexture  to add bet coins on stage using pixi v 6.1. 1. That was worked fine.
Now I updated to pixi v7.0.5. But now I unable to add coins like before. 
I attached the screenshot of my code. let me know how to fix this issue. 
![renderer-query](https://user-images.githubusercontent.com/72625004/209513073-5274f9a2-533e-406e-bc49-b5ac86a123bc.png)

**Comment by bigtimebuddy** [comment]
RenderTexture no longer supports arguments (width, height, etc) you need to use the options object. 

It's recommended to upgrade to the latest v6.5 so you get depreciation warnings for these things. See https://github.com/pixijs/pixijs/wiki/v7-Migration-Guide

**Reply by ajith-1998** [reply]
Thank You,
I have replaced the code as following,
 from 
 **app.renderer.render(oSprite, bet_rt)**
to 
**app.renderer.render(oSprite, { renderTexture: bet_rt })** 
and then it worked for me.
